src/main/java/org/smartregister/pathevaluator/PathEvaluatorLibrary.java

```diff
@@ -70,6 +70,10 @@ private PathEvaluatorLibrary(LocationDao locationDao, ClientDao clientDao, TaskD
 	public static void init(LocationDao locationDao, ClientDao clientDao, TaskDao taskDao, EventDao eventDao, StockDao stockDao) {
 		instance = new PathEvaluatorLibrary(locationDao, clientDao, taskDao, eventDao, stockDao);
 	}
+
+	public static void init(LocationDao locationDao, ClientDao clientDao, TaskDao taskDao, EventDao eventDao) {
+		instance = new PathEvaluatorLibrary(locationDao, clientDao, taskDao, eventDao, null);
```

Comment:
this should be calling init with null as last param

Reply:
changing